test/index.test-d.ts

```
@@ -28,6 +29,26 @@ expectType<string>(app.config.watcher.eventSources.default);
+expectType<MiddlewareFunc>(app.jsonp());
+expectType<MiddlewareFunc>(app.jsonp({ callback: 'callback' }));
+
+// i18n plugin types
+expectType<boolean>(app.config.i18n.writeCookie);
+expectType<string>(app.config.i18n.defaultLocale);
+expectType<string>(app.gettext('en-us', 'email'));
+expectType<boolean>(app.isSupportLocale('en-us'));
+expectType<string>(ctx.__('email'));
+expectType<string>(ctx.gettext('email %s', 'fengmk2'));
+expectType<string>(ctx.locale);
+expectType<string>(ctx.locale = 'en-us');
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)